src/main/resources/localization/economy.json

```diff
@@ -10,18 +10,21 @@
     "Your recent novel becomes a hit, earning you {amount} in royalties.",
     "You find a rare fossil on the beach and sell it to the museum for {amount}.",
     "You invested in the right stocks a the right time, earning {amount}.",
-    "Your NFT collection surges in price, so you sell them for {amount}."
+    "Your NFT collection surges in price, so you sell them for {amount}.",
+    "You finished digging in Patrick's mines for the day, He pay's you {amount}."
```

Comment:
Remove random names when referring to posession unless they are characters or public figures. "You finished digging in the mines for the day and make {amount}.

another alternative: "You finished digging in The Seven Dwarf's mine for the day. They pay you {amount}."

You get the idea